samples/NativeModuleSample/yarn.lock

```diff
@@ -0,0 +1,4584 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This file shouldn't exist, `samples/NativeModuleSample/` is not itself a node project.